_posts/2020-02-06-bazel-2.1.md

```diff
@@ -0,0 +1,85 @@
+---
+layout: posts
+title: "Bazel 1.1"
+authors:
+  - laurentlb
+---
+
+[Bazel 2.1](https://github.com/bazelbuild/bazel/releases/tag/2.1.0) has just been released.
+
+Bazel 2.1 is intended to be backward-compatible with Bazel 2.0. However it contains a single incompatible change that was deemed a bug fix and thus eligible for inclusion. We haven’t found any code broken by this change, but please report if it causes any burden to you.
+
+
+## Incompatible change
+
+*   The following attributes of `CcToolchainProvider`, which formerly were
+    accessible both as fields `x.f` and as methods `x.f()`, are now only fields:
+    `ld_executable`, `objcopy_executable`, `compiler_executable`,
+    `preprocessor_executable`, `nm_executable`, `objdump_executable`,
+    `ar_executable`, `strip_executable`.
+
+## Starlark
+
+*   `x.f()` is now equivalent to `y=x.f; y()`. That is, `x.f` should return the
+    same attribute value regardless of whether it is accessed as a field or
+    called like a method. Any arguments to the call are evaluated after the
+    `x.f` operation.
+*   Keyword-only arguments are now allowed:
+```python
+def foo(a, *, k): pass
+def foo(a, *b, k): pass
+foo(5, k = 6) # k can be specified only by keyword
+```
+*   Adds [`ctx.target_platform_has_constraint`](https://docs.bazel.build/versions/master/skylark/lib/ctx.html#target_platform_has_constraint) to allow rules to check the target platform's constraints.
+
+
+## bazelignore
+
+
+
+*   Similar to the
+    [.bazelignore](https://docs.bazel.build/versions/master/guide.html#.bazelignore)
+    in the main repository, a .bazelignore file in external repository will
+    cause the specified directories to be ignored by Bazel. Bazel won't try to
+    identify any packages under the directories, but the files can still be
+    referenced in other BUILD files.
+*   bazelignore files now support line comments, e.g. `# this is a comment`.
+
+
+## Android
+
+
+
+*   Experimental support for d8 merger is now available for use using
+    `--define=android_dexmerger_tool=d8_dexmerger`.
+*   Add desugaring support for `RoleManager#addRoleHolderAsUser`.
```

Comment:
```suggestion
*   Added desugaring support for `RoleManager#addRoleHolderAsUser`.
```

Reply:
Serge's recommendation was to use present tense (https://www.bazel.build/release-notes.html#guidelines).